tests/sles4sap/publiccloud/general_patch_and_reboot.pm

```diff
@@ -0,0 +1,47 @@
+# SUSE's openQA tests
+#
+# Copyright 2019 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: zypper
+# Summary: Refresh repositories, apply patches and reboot
+#
+# Maintainer: qa-c <qa-c@suse.de>
+
+use Mojo::Base 'publiccloud::basetest';
+use registration;
+use warnings;
+use testapi;
+use strict;
+use utils;
+use publiccloud::ssh_interactive qw(select_host_console);
+use publiccloud::utils qw(kill_packagekit);
+
+sub run {
+    my ($self, $run_args) = @_;
+    select_host_console();    # select console on the host, not the PC instance
+
+    foreach my $instance (@{$run_args->{instances}}) {
+        next if ($instance->{'instance_id'} !~ m/vmhana/);
+        record_info("$instance");
```

Comment:
Suggestion:
```
record_info($instance, 'Patching ' . $instance->{instance_id} ' / ' . $instance->public_ip);
```
Since the test module is not enabled in this PR, this change can be applied in an upcoming PR.